app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -2701,12 +2706,19 @@ export class Dispatcher {
       case CherryPickResult.ConflictsEncountered:
         this.startConflictCherryPickFlow(repository)
         break
+      case CherryPickResult.UnableToStart:
+        // This is an expected error such as not being able to checkout the
+        // target branch which means the cherry pick operation never started or
+        // was cleanly aborted.
+        this.appStore._endCherryPickFlow(repository)
+        return
```

Comment:
Nit: maybe use `break` here for consistency?
```suggestion
        break
```